cmd/rotate_certs.go

```diff
@@ -389,11 +403,11 @@ func (rcc *rotateCertsCmd) rotateEtcd(ctx context.Context) error {
 		}
 	}
 
-	log.Infoln("Rebooting all nodes... This might take a few minutes")
-	err := rcc.rebootAllNodes(ctx)
-	if err != nil {
-		return errors.Wrap(err, "rebooting the nodes")
-	}
+	// log.Infoln("Rebooting all nodes... This might take a few minutes")
```

Comment:
If it is no longer of use, better to remove the code rather than just comment it out. We'll always have the git history.